hydromt/gis/raster.py

```diff
@@ -2290,9 +2290,9 @@ def interpolate_na(
         kwargs.update(dict(method=method, extrapolate=extrapolate))
         if dim0:
             interp_data = np.empty(self._obj.shape, dtype=self._obj.dtype)
-            for i, (_, sub_xds) in enumerate(self._obj.groupby(dim0)):
+            for i, (_, sub_xds) in enumerate(self._obj.groupby(dim0, squeeze=False)):
```

Comment:
What is this doing?

Reply:
Squeeze removes empty dimensions. Currently the default is that it sill squeeze, but according to the error, this is being removed, so instead I set squeeze to False (which is the only way to silence the warning) and call squeeze on the result to maintain the same behavior.